apps/dalmatiner_db/src/dalmatiner_db_app.erl

```diff
@@ -22,8 +22,11 @@ start(_StartType, _StartArgs) ->
                     _ ->
                         100
                 end,
+    MaxConnections = application:get_env(dalmatiner_db, tcp_max_connections, 1024),
```

Comment:
Line 25 is too long:     MaxConnections = application:get_env(dalmatiner_db, tcp_max_connections, 1024),.